release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1300,6 +1300,12 @@ The Amazon Web Services (AWS) Elastic File System (EFS) Technology Preview featu
 
 * Technology preview badges were displayed on the OpenShift Pipelines workflows in the web console, though OpenShift Pipelines was GA. The Technology preview badges are now removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1945153[*BZ#1945153*])
 
+* Previously, pipelines created in the Git import flow for private repositories failed to run. This happened because the pipeline `ServiceAccount` did not use secrets created by the Git import flow for private Git repositories. With this update, you can add a secret name to the annotations of the pipeline `ServiceAccount` object, and add pipeline-specific annotations to the provided secret. As a result, pipelines for private Git repositories run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970485[BZ#1970485])
+
+* Previously, the Start Modal field did not recognize an empty string as a valid entry, so you had to enter a value. This happened even though the Tekton Operator recognizes empty strings as a valid `parameter default`. With this update, you have the option to enter an empty string. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966275[BZ#1966275])
```

Comment:
Like my previous comment, I think it's best to avoid `this, that, it, there` as subjects of sentences because these words are generic. 

`This failure/error/lack of recognition happened even though the Tekton Operator...`

Reply:
s/Start Pipeline modal
s/ Tekton Operator/OpenShift Pipelines Operator

Reply:
Done